src/index.js

```diff
@@ -21,11 +23,17 @@ getData().then((data) => {
           </div>
           <div class="card-footer">
             <button type="button" class="comment">Comments<button>
-            <button type="button" id="Reserve">Reservations</button>
+            <button class = "Reserve btn btn-primary">Reservations</button>
```

Comment:
Consider removing the space between the equality sign in line with standard tag attribute (`class="Reserve btn btn-primary"`)